app/views/rooms/_building_card.html.erb

```diff
@@ -19,6 +19,13 @@
         <%= building.address.titleize %>
       </div>
     <% end %>
+  
+  <% if building.notes.alert.present? %>
+    <div class="bg-yellow-550 text-sm px-2 mb-1 flex items-center space-x-2">
+      <%= svg('alert') %>
+        <span>Alert</span>
+    </div>
```

Comment:
Move line 23 thru 28 to below line 29 (AKA outside of the div) so the Alert is not underlined and misconstrued as a clickable link

Reply:
Please look at it again. I moved Alert our of that div